tests/src/JIT/Directed/Misc/gettype/gettypetypeofmatrix.csproj

```diff
@@ -22,8 +22,7 @@
     </CodeAnalysisDependentAssemblyPaths>
   </ItemGroup>
   <PropertyGroup>
-    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
-    <DebugType></DebugType>
+    <DebugType>None</DebugType>
```

Comment:
We need to understand the root cause of the difference here, i.e. whether it's some difference in the .targets files used or a difference in the tools themselves.  It's unlikely we'll want to change the DebugType attribute for any of the tests since the current settings were consciously chosen based on what an empty value meant.